### ISSUE TITLE
[Feature]: MoViNet-A6 Model

### ISSUE THREAD

**Issue by Hou9612** [towhee-io/towhee#2496]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Is your feature request related to a problem? Please describe.

Thanks for this wonderful project!  Code of MoViNet-A0~A5 have already in the project, will you release MoViNet-A6 in the future (The largest model of MoViNet series)?

**Comment by xiaofan-luan** [comment]
/assign @jaelgu 
any idea

**Comment by jaelgu** [comment]
@Hou9612 According to the paper, you can easily ensemble MoviNets A4&A5 to get A6. To get inference results of A6, just average logits by A4&A5.

**Comment by Hou9612** [comment]
> @Hou9612 According to the paper, you can easily ensemble MoviNets A4&A5 to get A6. To get inference results of A6, just average logits by A4&A5.

Got it! I will read the paper again and try it, thanks very much for your reply.
